lib/elixir/test/elixir/fixtures/parallel_struct/bar.ex

```diff
@@ -1,3 +1,3 @@
-defmodule Bar do
+defmodule :bar do
```

Comment:
That works, I originally wrote a test like that, but wondered if modifying an existing one would be preferable or not.